README.md

```diff
@@ -1 +1,53 @@
-# awesomeBooks
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Portfolio template
+
+
+## Built With
+
+- HTML, CSS
+- Github
+
+## Getting Started
+### To launch the project locally:
+#### Step 1:
+- Clone this repo locally using git in the command line and the following command
+ ```bash
+ > git clone 
+ ```
+- Alternatively, you can just download the complete zip file and extract the folder in your directory
+#### Step 2:
+- In the project directory, you can run:
+```bash
+> cd portfolio
+> code .
+```
+- Then open index.html in a live server.
+
+## Online version
+[Live-Demo]()
```

Comment:
**REQUIRED**

Add the complete and correct setup instructions including the necessary commands. This enables all users to have a more straightforward workaround for your project.

Also, You may choose to remove the live demo link since your project is not deployed yet, or you may add the correct Live link of your deployed app. Currently, the live link does not lead to a deployed version of your app.